src/jinja2/nodes.py

```diff
@@ -339,7 +339,7 @@ class With(Stmt):
 class Block(Stmt):
     """A node that represents a block."""
 
-    fields = ("name", "body", "scoped")
+    fields = ("name", "body", "scoped", "required")
```

Comment:
Need to document this attribute in the template docs.

Reply:
I just added a section titled `Required Blocks` under `Block Nesting and Scope`. Was this the right place to put it?